kyo-reactive-streams/shared/src/test/scala/kyo/interop/flow/PublisherToSubscriberTest.scala

```diff
@@ -225,17 +207,13 @@ abstract private class PublisherToSubscriberTest extends Test:
                             publisher.subscribe(subscriber3)
                             publisher.subscribe(subscriber4)
                         }
-                        .andThen(Async.sleep(2.seconds))
+                        .andThen(Async.sleep(1.seconds))
```

Comment:
could it be 100ms?

Reply:
This is more like a keep alive effect, the whole fiber will be interrupted below in 10.millis. So i dont think duration here is that important

Reply:
Is there a `Async.forever` or `Async.never` which will sleep forever or never return?